payment_paypal/indico_payment_paypal/plugin.py

```diff
@@ -46,7 +55,9 @@ class PaypalPaymentPlugin(PaymentPluginMixin, IndicoPlugin):
                         'business': ''}
     default_event_settings = {'enabled': False,
                               'method_name': None,
-                              'business': None}
+                              'business': None,
+                              'paypal_fixed_fee': "0.0",
+                              'paypal_percent_fee': "0%"}
```

Comment:
again here, these should be decimals
```suggestion
                              'paypal_fixed_fee': Decimal('0.0'),
                              'paypal_percent_fee': Decimal('0.0')}
```

Reply:
done